packages/desktop-ui-lib/src/lib/permission-manager/permission-manager.component.html

```
@@ -147,54 +149,60 @@ <h6>{{ 'TIMER_TRACKER.PERMISSIONS.ACCESSIBILITY' | translate }}</h6>
-								{{ 'TIMER_TRACKER.PERMISSIONS.OPEN_SYSTEM_SETTINGS' | translate }}
-							</button>
+						<div class="action-buttons">
+							@if (isMac()) {
+								<button nbButton status="primary" size="small" (click)="openAccessibilitySettings()">
+									<nb-icon icon="settings-2-outline"></nb-icon>
+									{{ 'TIMER_TRACKER.PERMISSIONS.OPEN_SYSTEM_SETTINGS' | translate }}
+								</button>
+							}
+
 							<button
 								nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)